tracer/contrib/gin-gonic/gintrace/gintrace_test.go

```diff
@@ -191,23 +191,25 @@ func TestGetSpanNotInstrumented(t *testing.T) {
 	assert.Equal(response.StatusCode, 200)
 }
 
-func getTestTracer(transport tracer.Transport) *tracer.Tracer {
-	testTracer := tracer.NewTracerTransport(transport)
-	return testTracer
+// getTestTracer returns a Tracer with a DummyTransport
```

Comment:
we may write a kind of ``utils`` package that hosts all common testing codes. In the meantime I've only fixed what is already here.